apps/aecore/src/aec_peers.erl

```diff
@@ -814,19 +831,19 @@ next_connect_delay(State) ->
     end.
 
 %% Gives the node some time to receive peers and establish outbound connections
-%% before it starts TCP probes of unverified peers.
+%% before it starts TCP probes of verified and unverified peers.
 -spec initial_tcp_probe_delay() -> non_neg_integer().
 initial_tcp_probe_delay() ->
     floor(ping_interval() * 1.5).
 
--spec next_tcp_probe_delay(state()) -> non_neg_integer().
-next_tcp_probe_delay(State) ->
-    %% TODO: this probably needs some tweaks, it should be also dependent on the
-    %% number of available unverified peers, number of verified peers vs
-    %% max_outbound, and maybe it could start to be more frequent when there are
-    %% more timeouts while connecting and the number of outbound connections is
-    %% low, etc.
-    #state{ last_tcp_probe_time = LastTime, outbound = Outbound } = State,
+-spec next_tcp_probe_delay(peer_pool_name(), state()) -> non_neg_integer().
+next_tcp_probe_delay(PeerPoolName, State) ->
+    LastTime =
```

Comment:
Could be its own function.

Reply:
Since it is only used in this small function, I'd rather keep it self contained in it.